.github/workflows/build-push-private.yml

```
@@ -107,20 +107,17 @@ jobs:
     name: Run Index Preparation
     needs: read-config
     secrets: inherit
-    uses: RMI-PACTA/workflow.prepare.pacta.indices/.github/workflows/run-index-preparation.yml@main
+    uses: RMI-PACTA/workflow.prepare.pacta.indices/.github/workflows/run-index-preparation.yml@build/103-check-existing-outputs
```

```suggestion
    uses: RMI-PACTA/workflow.prepare.pacta.indices/.github/workflows/run-index-preparation.yml@main
```

Putting this PR up for review, but this line needs to be altered before merging, but after https://github.com/RMI-PACTA/workflow.prepare.pacta.indices/pull/105 is closed.